cdc/processor/sinkmanager/manager.go

```diff
@@ -562,9 +565,10 @@ func (m *SinkManager) GetTableStats(tableID model.TableID) (pipeline.Stats, erro
 	}
 	checkpointTs := tableSink.(*tableSinkWrapper).getCheckpointTs()
 	m.memQuota.release(tableID, checkpointTs)
+	resolvedMark := checkpointTs.ResolvedMark()
```

Comment:
what's the difference between the resolved mark and checkpoints

Reply:
If it is a batch mode resolved ts it will be `TS-1`, otherwise, it will be `TS`.